ra/ra.go

```diff
@@ -1394,6 +1390,11 @@ func (ra *RegistrationAuthorityImpl) checkLimits(ctx context.Context, names []st
 			return err
 		}
 	}
+
+	// Check if there is rate limit space for a new order within the current window
+	if err := ra.checkNewOrdersPerAccountLimit(ctx, regID); err != nil {
+		return err
+	}
```

Comment:
Moving this to `checkLimits` means the ACME v2 new order rate limit is going to be applied for ACME v1 new certificate requests which seems surprising and maybe not correct. Is that what you intended?

Reply:
Good point. That was not what I intended. Reverted.